src/actions/authActions.js

```diff
@@ -120,12 +119,9 @@ export const loginAction = (pin: string, touchID?: boolean = false, onLoginSucce
         const { oAuthTokens: { data: OAuthTokensObject } } = getState();
         await dispatch(signalInitAction({ ...signalCredentials, ...OAuthTokensObject }));
         user = merge({}, user, userInfo);
-        if (generateNewConnKeys) {
-          await dispatch(updateConnectionKeyPairs(wallet.mnemonic, wallet.privateKey, user.walletId));
-        } else {
-          dispatch(updateConnectionsAction(user.walletId));
-        }
         dispatch(saveDbAction('user', { user }, true));
+        await
+        dispatch(updateConnectionKeyPairs(wallet.mnemonic, wallet.privateKey, user.walletId, generateNewConnKeys));
```

Comment:
During the biometrics login, you won't get a mnemonic and private key

Reply:
It's better to wrap this call with `if(generateNewConnKeys)..` or you need to change the method's signature and make the first and second params optional

Reply:
It is fine as long as the generation flow is wrapped in a condition with the last bool parameter 'generateNewConnKeys'

Reply:
Added changes in f0ac452